package.json

```diff
@@ -34,14 +34,14 @@
     "@emotion/react": "^11.6.0",
     "@emotion/styled": "^11.6.0",
     "classnames": "^2.3.1",
-    "cypress": "^9.0.0",
     "focus-visible": "^5.2.0"
   },
   "publishConfig": {
     "cache": "cache/.npm",
     "registry": "https://registry.npmjs.org/"
   },
   "devDependencies": {
+    "cypress": "^9.0.0",
```

Comment:
No yarn.lock/paclage-lock changes? Was `yarn install` / `npm install` ran?

Reply:
no changes to yarn.lock. the lock should already have a ref to cypress 9.0. do you see any issues here?

Reply:
Nope, all good 👍🏻